src/routes/users.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import auth from '../middleware/auth';
+import userController from '../controllers/users/signup'
+
+const router = express.Router();
+
+router.post('/api/signup', userController.signup);
```

Comment:
I would rather we logically group routes with the same general pattern in one file and in the server setup file we declare the general url pattern as a prefix with app.use